### ISSUE TITLE
Make OS base version indicator more obvious

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2668]
We have had a number of reports indicating our OS version 'mouse-over' is not accessible / lacks affordance. It is proposed that we remove this mouse-over and instead indicate our base OS version directly in the header. Hopefully then removing the potential confusion with the Linux kernel version.

Current state example:
![uses-openSUSE-mouse_over_example](https://github.com/rockstor/rockstor-core/assets/2521585/bcf3120a-3e60-4f8c-8d2e-40047dec2a37)

Proposed text would then  read:

**Uses openSUSE Leap: 15.5 Linux: ...**

**Comment by Hooverdan96** [comment]
https://github.com/rockstor/rockstor-core/blob/4d60a2c683140aaa10a8e5b201157163cc9a898b/src/rockstor/storageadmin/static/storageadmin/js/router.js#L980-L985

possibly could be as simple as changing this to:
```
 var distroInfo = function(data) { 
     $('#distro-info').text("Uses " + data.distro + ": " + data.version); 
     $('#distro-info').attr('title', data.version); 
 }; 
```
and either leave the mouse-over or get rid of it entirely. For Tumbleweed installs the version is a bit longer, given that it's comprised of `yyymmdd`:

<img width="134" alt="image" src="https://github.com/rockstor/rockstor-core/assets/35113775/f519cbfe-3cd2-45e6-9b34-91b59d5e67d2">

**Comment by phillxnet** [comment]
@Hooverdan96 Yes, that was what I was hoping.

> and either leave the mouse-over or get rid of it entirely.

I say the mouse over would then be redundant, so we simply drop it. I also don't think the long Tumbleweed version is a problem myself.

This obscure version surfacing (or lack there of) has come up as a failing a number of times now so I think it would be good to get this sorted during the current testing cycle.

**Comment by Hooverdan96** [comment]
I created a pull request. I did not test it with a TW build though, which I hope is ok.

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2670
